app/src/test/java/com/kickstarter/viewmodels/AddOnsViewModelTest.kt

```diff
@@ -58,18 +58,17 @@ class AddOnsViewModelTest : KSRobolectricTestCase() {
         viewModel.userRewardSelection(reward)
 
         assertEquals(
-            uiState.last(),
-            AddOnsUIState(
-                shippingSelectorIsGone = true
-            )
```

Comment:
This `test_hide_location_selection_on_reward_not_shippable` test and `test_show_location_selection_on_reward_is_shippable` are practically identical aside from the shipping type, but this test somehow does not pass, so I ended up comparing the specific shippingSelectorIsGone field instead.